CMakePresets.json

```diff
@@ -39,8 +21,7 @@
 			"architecture": {
 				"strategy": "external",
 				"value": "x64"
-			},
-			"toolchainFile": "$env{VCPKG_ROOT}/scripts/buildsystems/vcpkg.cmake"
```

Comment:
I think you still need that to load the toolchain, especially locally

Reply:
Is there a reason we don't set the toolchain in the presets? I see that now `VCPKG_TOOLCHAIN_FILE` is set by us in CMakeLists, can't we keep it in the toolchain anyways?